types/chrome/index.d.ts

```diff
@@ -5473,7 +5461,32 @@ declare namespace chrome.notifications {
          * @since Chrome 70
          */
         silent?: boolean | undefined;
-    }
+    } & (T extends true ? {
+        /**
+         * A URL to the sender's avatar, app icon, or a thumbnail for image notifications.
+         * URLs can be a data URL, a blob URL, or a URL relative to a resource within this extension's .crx file. Required for notifications.create method.
+         */
+        iconUrl: string;
+        /** Main notification content. Required for notifications.create method. */
+        message: string;
+        /** Which type of notification to display. Required for notifications.create method. */
+        type: string;
```

Comment:
PR looks good! I think the request in the [comment]( https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/57548#discussioncomment-1769973) to make this stricter to match the [TemplateType](https://developer.chrome.com/docs/extensions/reference/notifications/#type-TemplateType) in the docs would be good.